packages/design-system/src/components/N8nCallout/Callout.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+	<div :class="classes" role="alert">
+
+		<div :class="$style['message-section']">
+			<div :class="$style.icon">
+				<n8n-icon
+					v-bind="$attrs"
+					:icon="getIcon"
+					:size="theme === 'secondary' ? 'medium' : 'large'"
+				/>
+			</div>
+			<n8n-text
+				size="small"
+				v-bind="$attrs"
+				:bold="theme === 'secondary'"
+			>
+				{{ message }}
+			</n8n-text>
+			<span
+				v-if="actionText"
+				:class="$style['action-text']"
+				@click="$emit('action-text-click')"
+			>
+				{{ actionText }}
+			</span>
+		</div>
+
+		<n8n-link
+			v-if="trailingLinkText && trailingLinkUrl"
+			:to="trailingLinkUrl"
+			size="small"
+			theme="secondary"
+			:bold="true"
+			:underline="true"
+		>
+			{{ trailingLinkText }}
+		</n8n-link>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import N8nIcon from '../N8nIcon';
+import N8nLink from '../N8nLink';
+import N8nText from '../N8nText';
+
+export default Vue.extend({
+	name: 'n8n-callout',
+	components: {
+		N8nIcon,
+		N8nLink,
+		N8nText
+	},
+	props: {
+		theme: {
+			type: String,
+			required: true,
+			validator: (value: string): boolean =>
+				['info', 'success', 'secondary', 'warning', 'danger', 'custom'].includes(value),
+		},
+		icon: {
+			type: String,
+			default: 'info-circle'
+		},
+		message: {
+			type: String,
+			required: true
+		},
+		actionText: {
+			type: String,
+		},
+		trailingLinkText: {
+			type: String,
+		},
+		trailingLinkUrl: {
+			type: String,
+		},
+	},
+	data() {
+		return {
+			defaultIcons: {
+				'info': 'info-circle',
+				'success': 'check-circle',
+				'warning': 'exclamation-triangle',
+				'danger': 'times-circle',
+			}
```

Comment:
Should keep this outside the component because it doesn't need to become reactive.